src/twisted/topfiles/8764.feature

```diff
@@ -0,0 +1 @@
+Twisted is now versioned based on a year.month.patch basis.
```

Comment:
Can you elaborate a little bit more on this? Ie. what was it before? What's the improvement?